Modules/FindOpenHaptics.cmake

```diff
@@ -0,0 +1,320 @@
+# - try to find OpenHaptics libraries
+#
+# Cache Variables: (probably not for direct use in your scripts)
+#  HDAPI_INCLUDE_DIR
+#  HDAPI_LIBRARY
+#  HDAPI_LIBRARY_RELEASE
+#  HDAPI_LIBRARY_DEBUG
+#  HDAPI_HDU_INCLUDE_DIR
+#  HDAPI_HDU_LIBRARY
+#  HDAPI_HDU_LIBRARY_RELEASE
+#  HDAPI_HDU_LIBRARY_DEBUG
+#  HLAPI_INCLUDE_DIR
+#  HLAPI_LIBRARY
+#  HLAPI_LIBRARY_RELEASE
+#  HLAPI_LIBRARY_DEBUG
+#  HLAPI_HLU_INCLUDE_DIR
+#  HLAPI_HLU_LIBRARY
+#  HLAPI_HLU_LIBRARY_RELEASE
+#  HLAPI_HLU_LIBRARY_DEBUG
+#
+# Non-cache variables you might use in your CMakeLists.txt:
+#  OPENHAPTICS_FOUND
+#  HDAPI_INCLUDE_DIRS
+#  HDAPI_LIBRARIES
+#  HDAPI_HDU_INCLUDE_DIRS
+#  HDAPI_HDU_LIBRARIES
+#  HLAPI_INCLUDE_DIRS
+#  HLAPI_LIBRARIES
+#  HLAPI_HLU_INCLUDE_DIRS
+#  HLAPI_HLU_LIBRARIES
+#  OPENHAPTICS_LIBRARIES - includes HD, HDU, HL, HLU
+#  OPENHAPTICS_INCLUDE_DIRS
+#  OPENHAPTICS_BINARY_DIR
+#  OPENHAPTICS_UTILITIES_BINARY_DIR
+#
+# Requires these CMake modules:
+#  CleanDirectoryList
+#  CleanLibraryList
+#  ListCombinations
+#  ProgramFilesGlob
+#  SelectLibraryConfigurations (included with CMake >=2.8.0)
+#  FindPackageHandleStandardArgs (known included with CMake >=2.6.2)
+#  CMake 2.6.3 (uses "unset")
+#
+# Original Author:
+# 2009-2012 Ryan Pavlik <rpavlik@iastate.edu> <abiryan@ryand.net>
+# http://academic.cleardefinition.com
+# Iowa State University HCI Graduate Program/VRAC
+#
+# Copyright Iowa State University 2009-2010.
+# Distributed under the Boost Software License, Version 1.0.
+# (See accompanying file LICENSE_1_0.txt or copy at
+# http://www.boost.org/LICENSE_1_0.txt)
+
+cmake_minimum_required(VERSION 2.6.3)
+
+
+SET( OpenHaptics_PATH_HINTS
+  "C:/OpenHaptics/Developer/3.4.0"
+  )
+
+
+FIND_PATH( OpenHaptics_ROOT_DIR Geomagic_OpenHaptics.ico
```

Comment:
Does this file also exist on linux?

Reply:
I will double-check that, and add linux paths

Reply:
Let me know, and I'll add the fixes (Just paste them as a reply to this), as I've made changes in the master

Reply:
So, the linux sdk is structured very differently from the windows and so I will need to rewrite a bunch to support linux compiling.  Will get back to you when that is done.

Reply:
Well, honestly it's up to you. Linux support would be nice, but not necessary.